aten/src/ATen/detail/AcceleratorHooksInterface.h

```diff
@@ -50,6 +50,20 @@ struct TORCH_API AcceleratorHooksInterface {
     return -1;
   }
 
+  virtual c10::Stream getCurrentStream(DeviceIndex device) const {
+    TORCH_CHECK(false, "Backend doesn't support getCurrentStream()");
+    return c10::Stream::unpack3(-1, 0, c10::DeviceType::CPU);
+  }
+
+  virtual c10::Stream getDefaultStream(DeviceIndex device) const {
```

Comment:
Default stream may have an implicit synchronization semantic. Not every backend has this feature.

Reply:
@guangyey Really helpful! Will remove this implementation.